config/validator.go

```diff
@@ -60,25 +60,38 @@ func UniqueValuesInSliceValidator(v interface{}, param string) error {
 	return nil
 }
 
-// ClientClustersEntryLogicalValidator validate Client->Clusters entry and make sure that all required clusters are defined
-func (c *YamlConfig) ClientClustersEntryLogicalValidator(valid *bool, validationErrors *map[string][]error) {
-	errorsList := make(map[string][]error)
-
-	if len(c.Client.Clusters) == 0 {
-		*valid = false
-		errorDetail := []error{errors.New("Empty clusters definition")}
-		errorsList["ClientClustersEntryLogicalValidator"] = errorDetail
+//RegionsEntryLogicalValidator validator for Regions part
```

Comment:
Really? It validates "regions" part of configuration file... and tells if it's ok or not...